CMakeLists.txt

```diff
@@ -60,7 +60,8 @@ else (MSVC)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
     set(CMAKE_POSITION_INDEPENDENT_CODE ON)
     if (APPLE)
-        add_compile_options(-arch x86_64 -fno-common)
+#       add_compile_options( -arch x86_64 -arch arm64  -fno-common)
+        add_compile_options( -fno-common)
```

Comment:
Can you remove commented code? Maybe replace with a sentence saying why you don't need `-arch x86_64 -arch arm64`?